test/jdk/javax/accessibility/manual/ComboBoxDemo.html

```diff
@@ -72,4 +72,3 @@
 </body>
 </html>
 
-        
```

Comment:
The file contained trailing white-space at the last line, it is removed by the IDE.